olive/cli/base.py

```diff
@@ -38,132 +38,241 @@ def register_subcommand(parser: ArgumentParser):
     def run(self):
         raise NotImplementedError
 
+    @staticmethod
+    def _get_model_name_or_path(model_name_or_path) -> Union[str, Dict[str, str]]:
```

Comment:
This will be called in the child class, so it's not truly private to this class. Can we remove the underscore? Same for below.

Reply:
As per Python PEP [here](https://peps.python.org/pep-0008/#method-names-and-instance-variables)

_Use one leading underscore only for non-public methods and instance variables._

These functions are _protected_ i.e. can be used in sub-classes/derived classes but not be invoked outside.
Unfortunately, there is no recommendation to differentiate between protected and private member functions. However, some recommend using single underscore and double underscores.

Reply:
We don't need to make them @staticmethod.

Reply:
So, make them member functions or global? `self.xxxxx()` vs. `xxxx()`

Making them members doesn't make sense because there's no use of `self` in the implementation.
The issue with making them global is each of them have to be imported for use.

Reply:
Class methods just adds another unused argument, just like member functions (with self) do. I don't see any reason to change them to member functions or class functions, except for aesthetic reasons.

Logically, them being static is correct because they don't have any use of cls or self. Global just makes the importa look complicated.

Either way, @devang-ml recommended reverting all the _beautification_/_code organization_ changes and focusing the PR only on the quantize command. So, I will do that and report the PR.

Reply:
yes, I think we can revisit this later if needed. My suggestion was to make the register_subcommand into a class method so that calls like `CaptureOnnxGraphCommand._add_logging_options` could be replaced with `cls._add_logging_options`. for the rest staticmethod is cleaner. 

But it's mostly aesthetic as you said since we don't enforce a particular style for these cases.